ext/groonga/rb-grn-table.c

```diff
@@ -205,31 +205,33 @@ rb_grn_table_inspect (VALUE self)
 }
 
 /*
- * テーブルに名前が _name_ で型が _value_type_ のカラムを定義
- * し、新しく定義されたカラムを返す。
+ * This function defines a column taht name is _name_ and
+ * type is _value_type_.
+ * It returns new defined cokumn.
  *
  * @overload define_column(name, value_type, options={})
  *   @param options [::Hash] The name and value
  *     pairs. Omitted names are initialized as the default value.
  *   @option options :path
- *     カラムを保存するパス。
- *   @option options :persistent (永続カラム)
- *     +true+ を指定すると永続カラムとなる。省略した場合は永
- *     続カラムとなる。 +:path+ を省略した場合は自動的にパスが
- *     付加される。
+ *     A path for saving column.
+ *     If we omit +:path+ , add a path automatically.
+ *   @option options :persistent (A persistent column)
+ *     If we specify +true+, a column is a persistent column.
+ *     If we omit this option, a column is a persistent column.
```

Comment:
It seems that there is no difference `true` and `omit` case in this description.